src/pages/ReportDetailsPage.js

```diff
@@ -57,43 +56,60 @@ class ReportDetailsPage extends Component {
     constructor(props) {
         super(props);
 
-        this.menuItems = [];
+        this.renderOptions = this.renderOptions.bind(this);
+    }
+
+    renderOptions() {
```

Comment:
This was necessary since before, the options were getting defined in the constructor. This worked before because nothing the options relied on could chance.

Now that new users can join the room, the number of participants can change, therefore the Members options needs to be able to re-render to account for this change.